config.go

```diff
@@ -15,90 +14,24 @@ import (
 const configName = ".phraseapp.yml"
 const defaultDir = "./"
 
-type Credentials struct {
-	phraseapp.Credentials
-	Username string `cli:"opt --username -u desc='username used for authentication'"`
-	Token    string `cli:"opt --access-token -t desc='access token used for authentication'"`
-	TFA      bool   `cli:"opt --tfa desc='use Two-Factor Authentication'"`
-	Host     string `cli:"opt --host desc='Host to send Request to'"`
-	Debug    bool   `cli:"opt --verbose -v desc='Verbose output'"`
-}
-
-func ConfigCallArgs() (map[string]string, error) {
+func ReadConfig() (*phraseapp.Config, error) {
 	content, err := ConfigContent()
 	if err != nil {
-		content = "{}"
+		content = []byte("{}")
```

Comment:
we should at least log the error coming from ConfigContent() because so many issues can occur here